burn-core/src/grad_clipper/base.rs

```diff
@@ -0,0 +1,69 @@
+use burn_tensor::{backend::Backend};
+use crate::tensor::Tensor;
+/// Gradient Clipper provides a way to mitigate the exploding gradients
+/// problem by clipping every component of the gradient to a value during 
+/// backpropagation.
+pub struct GradientClipper {
+    clip_value: Option<f32>,
+    clip_norm: Option<f32>,
+}
```

Comment:
I think an enum is better in this case.

Reply:
Is there a reason you kept the option?

Reply:
@agelas

Reply:
@nathanielsimard My bad, had to go back to using a  `struct` when I thought about making it a config to the optimizer. Should be fixed now.